src/styles/main.scss

```diff
@@ -6,6 +6,7 @@ html {
     font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
     font-size: 62.5%;
     font-weight: 300;
+    line-height: 1.42857143;
```

Comment:
It's 20/14.  It's a commonly used number by other frameworks as well, including Bootstrap.